odc/geo/ui.py

```diff
@@ -73,7 +76,11 @@ def svg_base_map(
     # in pixels
     min_span = min(40, sz)
 
-    if span_x > span_y:
+    if min(span_x, span_y) < 1e-4:
```

Comment:
Could you add a code comment explaining why `1e-4` is a fine constant? I _think_ that its because we're assumed to be in web mercator here, but I'm not sure.

Reply:
kinda combination of numeric precision, round number and "what is likely empty anyway", this is roughly 11m at equator, so 1 sentinel pixel or there about, even if we had high-res map this would be way too zoomed in to make sense, maybe we should pick a bigger number.

now that I re-read it, it's not quite right, I don't think this will display something like `1x10_000` slice of sentinel correctly. i think this was meant to be `max` not `min`. I'll change it and add comment about what 1e-4 implies here.

Reply:
reworked that and change tolerance check to be purely numeric based